### DISCUSSION TITLE
How are breaking changes currently marked?

### DISCUSSION THREAD

**Discussion by alma-esaniemi** [rnmapbox/maps#3315] [Q&A]
Hi! I noticed that the changelog file https://github.com/rnmapbox/maps/blob/main/CHANGELOG.md is no longer maintained and the releases list seems to usually offer limited info on what each release changes, apart from a seemingly generated commit lists.

I noticed upgrading from `10.0.11` to `10.1.5` broke imports for `@rnmapbox/maps/lib/typescript/types/OnPressEvent` and `@rnmapbox/maps/lib/typescript/components/Camera` as the files are now under `@rnmapbox/maps/lib/typescript/src`but found no warning about this.

**Comment by mfazekas** [comment]
@alma-esaniemi We don't expect you to do import other than:

```
import Mapbox from "@rnmapbox/maps"
```

or 

```
import { MapView, Camera, ... } from "@rnmapbox/maps"
```

Our file organisation and internal type names are subject to change even in patch releases. Those are not considered to be a breaking change.

**Reply by alma-esaniemi** [reply]
Hi @mfazekas and thanks for the answer! Is the place I'm importing types from wrong? I'd like to have typing for the event for custom on-press handlers and the like.

**Reply by mfazekas** [reply]
I don't think that we export that type.
We can export that or you can access it with something like:

```
Parameters<ComponentProps<typeof MapView>['onPress']>[0];
```
